src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -694,7 +694,7 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                windows_admin_username=None,
                windows_admin_password=None,
                kubernetes_version='',
-               node_vm_size="Standard_DS2_v2",
+               node_vm_size="Standard_D2s_v3",
```

Comment:
the change itself is ok. Is the default vm documented somewhere?

Reply:
@jluk , could you modify our documentation after the change is merged.